plugins/commands/push/command.rb

```diff
@@ -9,8 +9,17 @@ def self.synopsis
 
       # @todo support multiple strategies if requested by the community
       def execute
+        options = {}
+
         opts = OptionParser.new do |o|
           o.banner = "Usage: vagrant push [strategy] [options]"
+          o.separator ""
+          o.separator "Options:"
+          o.separator ""
+
+          o.on("-m", "--message BODY", "Text to identify this push") do |m|
+            options[:message] = m
+          end
```

Comment:
I also wonder if this should even be a top-level option, because what does it mean in the general sense?